sources/packages/forms/src/form-definitions/supportinguserspartner2022-2023.json

```diff
@@ -4239,9 +4242,263 @@
+                      "type": "content",
+                      "tags": [],
+                      "conditional": {
+                        "show": "",
+                        "when": "",
+                        "eq": ""
+                      },
+                      "properties": {},
+                      "customConditional": ""
+                    },
+                    {
+                      "label": "Collection and use of information",
+                      "attrs": [
+                        {
+                          "attr": "",
+                          "value": ""
+                        }
+                      ],
+                      "content": "<strong>Collection and use of information</strong>. The information included in this form and authorized above is collected under Sections 26c and 26e of the Freedom of Information and Protection of Privacy Act and under the authority of the Canada Student Financial Assistance Act, R.S.C. 1994, Chapter C-28 and StudentAid BC. The information provided will be used to determine eligibility for a benefit through StudentAid BC and for statistical and evaluation purposes. If you have any questions about the collection and use of this information, contact the Director, StudentAid BC, Ministry of Advanced Education and Skills Training, PO Box 9173, Stn Prov Govt, Victoria, BC V8W 9H7, call 1-800-561-1818 (toll-free in Canada/U.S.) or +1-778-309-4621 from outside North America.",
```

Comment:
Same here. https://github.com/bcgov/SIMS/pull/2631/files#r1440848816

Reply:
The format is updated to include collection and use of information inside the rich text editor. Please have a look.